keyboards/planck/keymaps/dvorak2space/keymap.c

```diff
@@ -0,0 +1,215 @@
+#include QMK_KEYBOARD_H
+#include "passwords.c" //Instead of extern just to cut down on compile time. Holds a single array.
```

Comment:
This may work better: https://github.com/qmk/qmk_firmware/tree/master/users/drashna#secret-macros

Reply:
Seems a bit excessive for a single secret, and you still need to make a blank array for each. Only advantage I see is not needing to exclude each secret — unless I'm missing something?

Reply:
It may be, but if you change branches or update your repo frequently, it may make it a pain to do it this way.  

The way I have outlined .... while it may be more work, it would  not have issues when updating or changing branches. 

But it's your keymap, so it's your call.

Reply:
Force pulling to fix anything overwrites it, so I see your point. This works, though, and as it's a bad password system anyway I'll probably be the only one using it.